spec/factories/resourcetypes.rb

```diff
@@ -0,0 +1,10 @@
+FactoryGirl.define do
+  factory :resourcetype do
+    name "MyString"
+femakind "MyString"
+femacode "MyString"
+status "MyString"
+description "MyText"
```

Comment:
Could you fix this indenting?  (Although, if you go with my advice about conventional compound word usage like "ResourceType", "fema_kind", etc, then this should resolve itself. :sunglasses: )